src/lib/fileContentHelper.js

```diff
@@ -4,7 +4,7 @@ const path = require('path');
 
 const fileContentHelper = {
   getFileContent(filePath, id) {
-    const fileUrl = path.resolve(__dirname, `../pages/docs/${filePath}/${id}.md`);
+    const fileUrl = path.resolve(`${filePath}/${id}.md`);
```

Comment:
use `join(filePath, `${id}.md`)` or don't use anything here. Btw we don't need a new variable declaration here.